gf-guide/diacritics.md

```diff
@@ -36,7 +36,7 @@ This guide will give users an overview introduction to diacritics both from a de
 - **In Latin script, it is expected that all marks share the same distance to the base letter.** For particular cases like script fonts where the x-height could vary, marks should at least appear to be at the same optical distance to the base letter.
 - **Idiosyncratic or decorative diacritics have given way to more universal or neutral forms.**
 - **Ensure creating all the combining marks required for the languages the font are supporting**, as well as the so-called "Legacy Marks". ([See below](#legacy--spacing-marks).) Google Fonts requires at least the GF Latin Core as the minimum set for a font addressing the Latin script. Please read more about our <a href="https://googlefonts.github.io/gf-guide/requirements.html#glyphsets">Glyphsets</a>.
-- **Usually, the design of the diacritics needs to be adjusted in size or slope to work better with the uppercase letters**. So ideally there should be at least two sets of diacritics: lowercase and uppercase marks. (e.g. `acutecomb` and `acutecomb.case`)
+- **Create a set of marks for capital letters** To properly display uppercase letters with diacritics, a separate set of `.case` marks should be created. These marks should be designed with adjustments to size, height, or slope and are typically wider, shorter, and flatter than their lowercase counterparts so they use less vertical space avoiding collisions with above text lines. These adjustments are particularly relevant for languages like Vietnamese that require stacked diacritics.
```

Comment:
Not a big deal, but To after letters** shouldn't be capitalized since the in no period before it

Reply:
Also maybe add a comma after vertical space since it's a long sentence and there is a natural break there: `...less vertical space, avoiding collisions...`

Not really needed, just makes reading the long sentence easier IMO.